src/utils.py

```diff
@@ -108,7 +108,7 @@ def do_CV(X,y, model, multi_class=True, test_size=0.3):
     if multi_class == False:
         my_dict = {'I':1, 'SR':-1}
         print("ROC AUC score")
-        print(roc_auc_score(np.vectorize(my_dict.get)(y_test), np.vectorize(my_dict.get)(y_pred)))
+        print(roc_auc_score(np.vectorize(my_dict.get)(y_test), model.predict_proba(X_test)[:, 1]))
```

Comment:
hmm we should also update plot_roc_binary to use the 1st column instead of 0th. Still not clear why one or the other though, I feel like it picks class randomly?